pkg/cinderbackup/statefulset.go

```diff
@@ -44,8 +44,8 @@ func StatefulSet(
 	rootUser := int64(0)
 	// Cinder's uid and gid magic numbers come from the 'cinder-user' in
 	// https://github.com/openstack/kolla/blob/master/kolla/common/users.py
```

Comment:
-1: Remove this from here.
nit: Move it to the constant definition as mentioned above.